### ISSUE TITLE
[New] Approve Thread for Disambiguated Aliases

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#8]
We've never landed on a clear consensus regarding disambiguated performer aliases, for example: `Jane (Studio)`. I know we have many of these in the database already, likely from scraping/copying from databases like IAFD that use this formatting. There have been recent conversations about expanding performer aliases to include this studio-based disambiguation as a built-in feature. Adding the studio in parentheses within the text isn't always known and would be a temporary measure unless/until a similar feature is added to StashDB instead. Confusion over how we handle these kinds of aliases pops up occasionally so we'll need to decide what our approach should be in the meantime.

**Comment by echo6ix** [comment]
I for one am not going to invest the time to add disambiguation for aliases, and I'd like to think I'm not a low-effort contributor. 

Reasons:

- It invites yet more guidelines for contributors to follow.

- Performer "aliases" are often not relegated to just one per studio; they often use the same alias for multiple studios. [Here is a mild example](https://www.indexxx.com/m/ryann-shields). Which studio gets to become the disambiguation for her aliases? And will excluding a studio for an alias confuse users who are searching for it and don't find it? Add one alias for every studio maybe? Add all the studios as a disambiguation for each name? It just adds more work, more confusion, more guidelines.

- It looks like Stash is possibly moving towards an alias-studio association based on WithoutPants' comments that it's not something he wants to jump into on the first iteration of the new alias paradigm. If this is in fact the case, and if we want to increase synergy between Stash and StashDB, manually adding studio disambiguation to aliases is not the way to do it. It will create an inconsistent mess.

TLDR. Less manual data entry not more. Do not manually add disambiguation to aliases. Wait for an automated solution.

**Comment by PhuriousGeorge** [comment]
The _only_ way I can perceive this to be successful would be to tie aliases to studios, but even there (for example) you have some studios calling 5+ performers "Jane" - how do you then distinguish even further?  Personally I think disambiguating aliases is almost an impossible task if not tied to studios and either way we should try to implement tools to help users find the performer they're looking for (like the feature request for a on-hover image of the performer).
